src/map/utils/jailutils.cpp

```diff
@@ -43,15 +40,11 @@ namespace jailutils
 
     void Add(CCharEntity* PChar)
     {
-        PChar->PAI->SetController(nullptr);
```

Comment:
This was the crux of the problem. Setting controller to null was harmless.... until it wasn't